### DISCUSSION TITLE
Search/sort option in the installed extension list?

### DISCUSSION THREAD

**Discussion by HubKing** [mjakeman/extension-manager#344] [Ideas]
Currently, it seems that the installed extensions list is simply alphabetical. When there are many extensions, it becomes increasingly difficult because there is nothing like an icon.

How about, adding a simple search (filter) bar above the list? And sort method, like the installation date/size. I guess that if the user has installed some extensions without using Extension Manager, maybe it is impossible to know the installation date, but at least for those extensions installed using E.M., you can track the installation date, can't you?

**Comment by mjakeman** [comment]
I plan to add a universal search as part of v1.0 (the next version) - see issue #8 for more.

For other filters, I'm not sure it's worth tracking installed size since extensions are rarely over a few kilobytes. I'd like to avoid having to have additional metadata in extension manager other than what's already available locally in the extension's manifest, so sorting by installed date is probably not on the cards as it would complicate things quite a bit.
